lib/modules/Client.js

```diff
@@ -7,6 +7,14 @@ import navigationMiddleware from './navigationMiddleware';
 import platform from './reducer';
 import actions from './actions';
 
+export const activateClient = () => async (dispatch, getState) => {
```

Comment:
Similarly to the initialShell, this is client-only; could go in clientActions.js, or here, or in actions.js.